### PR TITLE
Cake5 updates

### PR DESCRIPTION
**Pull request by ADmad**

### CODE REVIEW

File: src/Middleware/AuthorizationMiddleware.php [hunk 1]
```diff
@@ -78,23 +78,10 @@ class AuthorizationMiddleware implements MiddlewareInterface
...
-            !$subject instanceof AuthorizationServiceProviderInterface
-        ) {
-            $expected = implode('` or `', [
-                AuthorizationServiceInterface::class,
-                AuthorizationServiceProviderInterface::class,
-            ]);
-            $type = is_object($subject) ? get_class($subject) : gettype($subject);
-            $message = sprintf('Subject must be an instance of `%s`, `%s` given.', $expected, $type);
-
-            throw new InvalidArgumentException($message);
```
Comment (markstory) [inline src/Middleware/AuthorizationMiddleware.php:95]: :clap: So nice that the type system can take care of this now.

**Review by markstory — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -77,5 +78,7 @@
             "cakephp/plugin-installer": true,
             "dealerdirect/phpcodesniffer-composer-installer": true
         }
-    }
+    },
+    "minimum-stability": "dev",
```
Comment (othercorey) [inline composer.json:82]: Is this needed with explicit versions?

Comment (ADmad) [reply composer.json:82]: Yes it's needed to be able to install indirect non-stable dependencies. In this particular case `bake` has `twig-view` with `dev-cake5`.

Comment (othercorey) [reply composer.json:82]: Ah right.

**Pull request merged**